ts/javascript/initializers.ts

```diff
@@ -42,17 +46,28 @@ export const npmInitializer: Initializer = (dir) => {
             nodeVersion: "18",
           })
         `,
-        ...(scripts.includes("test")
-          ? ['.addCheck("test", "npm run test")']
-          : []),
-        ...nonTestScripts.map(
-          (name) =>
-            `.addExecutable(${JSON.stringify(name)}, ${JSON.stringify(
-              `npm run ${name}`,
-            )})`,
-        ),
-      ].join("\n  "),
+        ...(isViteProject ? [`.add(garn.javascript.vite)`] : []),
+        ...("test" in scripts ? ['.addCheck("test", "npm run test")'] : []),
+        ...nonTestScripts
+          .filter(([name, script]) =>
+            isViteProject ? !isSuperSeededByVitePlugin(name, script) : true,
+          )
```

Comment:
Nit, but maybe a single filter is more readable? TIOLI

```suggestion
        ...Object.entries(scripts)
          .filter(([name, script]) => {
            if (name === "test") return false;
            if (isViteProject && isSuperSeededByVitePlugin(name, script)) return false;
            return true;
          })
```

Then we can remove `nonTestScripts` above.